### PR TITLE
Ui/fix for ios

### PR DESCRIPTION
**Pull request by jiayingy**
Fixed double click compatibility issues on iOS browser

### CODE REVIEW

File: sashimi-webapp/src/components/file-manager/File.vue [hunk 1]
```diff
@@ -24,6 +24,16 @@
...
   props: ['file'],
   data() {
+    return {
+      onClickDetails: {
+        x: '',
+        y: '',
+        isClickInProgress: false
+      },
+      clearIsClick: () => {
+        this.isClickInProgress = false;
```
Comment (amoshydra) [inline sashimi-webapp/src/components/file-manager/File.vue:34]: `this.isClickInProgress` is not the same with `this.onClickDetails.isClickInProgress`.
Therefore, `this.onClickDetails.isClickInProgress` is never set to false.

This result in second tapping to always be executed as a double click event.

File: sashimi-webapp/src/components/file-manager/File.vue [hunk 2]
```diff
@@ -45,6 +55,18 @@ export default {
 
       this.file.rename(newFileName);
     },
+    onClick(event) {
+      if (this.onClickDetails.isClickInProgress && event.x === this.onClickDetails.x && event.y === this.onClickDetails.y) {
```
Comment (amoshydra) [inline sashimi-webapp/src/components/file-manager/File.vue]: `event.x === this.onClickDetails.x && event.y === this.onClickDetails.y` require the subsequent click to be in the exact same location.

This is very difficult for touch device as the finger is not likely to hit the exact coordinate.

You may either implement a threshold area or remove this check.

Threshold area e.g.
```javascript
dblClickCheck(event) {
  const threshold = 20;
  return (this.onClickDetails.isClickInProgress &&
          ((Math.abs(event.x - this.onClickDetails.x) < threshold) &&
            (Math.abs(event.y - this.onClickDetails.y) < threshold)
          ));
},
onClick(event) {
  if (this.dblClickCheck(event)) {
    this.openFile();
    this.clearIsClick();
    clearTimeout(this.clearIsClick);
  } else {
    this.onClickDetails.x = event.x;
    this.onClickDetails.y = event.y;
    this.onClickDetails.isClickInProgress = true;
    setTimeout(this.clearIsClick, 1000);
  }
},
```

I chose a threshold of 20 based on a quick experiment I conducted with my 5.5 inches screen Android phone.
Result showing data for 138 touches.

20px is equivalent to:

|Device|Screen Size (inches) |Actual (mm)|
---|---|---
Oneplus 3| 5.5 | 4.50
iPhone 5 | 5 | 3.47
Nexus 6P | 5.7 | 3.46
Nexus 5X | 5.2 | 3.16

**Review by amoshydra — CHANGES_REQUESTED** [review]

File: sashimi-webapp/src/components/file-manager/File.vue (hunk 1 above)
Comment (jiayingy) [reply sashimi-webapp/src/components/file-manager/File.vue:34]: Have replaced it with threshold instead of checking for exact coordinates

**Review by amoshydra — APPROVED** [review]
This should work. 👍

**Pull request merged**